Dockerfile

```diff
@@ -30,5 +30,14 @@ COPY --from=ksops-builder /go/bin/kustomize /usr/local/bin/kustomize
 # Copy the plugin to kustomize plugin path
 COPY --from=ksops-builder /go/src/github.com/viaduct-ai/kustomize-sops/*  $KUSTOMIZE_PLUGIN_PATH/viaduct.ai/v1/${PKG_NAME}/
 
+RUN apt-get update && \
+  apt-get install -y \
+    curl \
+    gpg && \
```

Comment:
gpg 2.2.12 is included in the argocd image

Reply:
oh whoops I missed this - it wasn't in the original argocd last I checked, but seems to be there now